content/strategy-goals/strategy/index.md

```diff
@@ -35,6 +35,8 @@ We have identified five key use cases for our universal code search platform:
 1. [Fixing security vulnerabilities](use-cases/fixing-security-vulnerabilities.md)
 1. [Incident Response](use-cases/incident-response.md)
 
+There is no explicit mention of developer velocity in these use case titles, because they all improve developer velocity.
```

Comment:
```suggestion
There is no explicit mention of developer velocity in these use case titles, because they all improve developer velocity. Notably, research shows that revenue growth among companies who invest in the velocity of their developers see revenue growth upto 4-5 times faster than those who don’t.
```